TransbankAutoservicioSDKConsoleExample/Program.cs

```diff
@@ -0,0 +1,348 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Transbank.POSAutoservicio;
+using Transbank.Responses.AutoservicioResponse;
+using Transbank.Responses.CommonResponses;
+
+namespace TransbankPosSDKConsoleExample
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string portName = SelectCom();
+            int baudrate = SelectBaudrate();
+
+
+            POSAutoservicio.Instance.OpenPort(portName, baudrate);
+            Console.WriteLine("\nSerial port opened successfully");
+            Console.WriteLine("Showing Menu");
+            int key = ShowMenu();
+            Console.WriteLine("\n");
+            bool print = false;
+            bool voucher = false;
+            int amount = 0;
+            string ticket = "";
+            do
+            {
+                switch (key)
+                {
+                    case 1:
```

Comment:
It's not necessary but I would appreciate a few comments on conditions which involve only numbers. What I mean is a comment saying the text of the menu item or a small explanation about it

Reply:
The showmenu function has the description of switch / case